gobblin-yarn/src/test/java/org/apache/gobblin/yarn/GobblinYarnAppLauncherTest.java

```diff
@@ -156,32 +172,27 @@ public void setUp() throws Exception {
         .getResource(GobblinYarnAppLauncherTest.class.getSimpleName() + ".conf");
     Assert.assertNotNull(url, "Could not find resource " + url);
 
-    this.config = ConfigFactory.parseURL(url)
-        .withValue("gobblin.cluster.zk.connection.string",
-                   ConfigValueFactory.fromAnyRef(testingZKServer.getConnectString()))
-        .resolve();
+    this.config = ConfigFactory.parseURL(url).withValue("gobblin.cluster.zk.connection.string", ConfigValueFactory.fromAnyRef(testingZKServer.getConnectString())).resolve();
 
     String zkConnectionString = this.config.getString(GobblinClusterConfigurationKeys.ZK_CONNECTION_STRING_KEY);
-    this.helixManager = HelixManagerFactory.getZKHelixManager(
-        this.config.getString(GobblinClusterConfigurationKeys.HELIX_CLUSTER_NAME_KEY), TestHelper.TEST_HELIX_INSTANCE_NAME,
-        InstanceType.CONTROLLER, zkConnectionString);
+    this.helixManager = HelixManagerFactory
+        .getZKHelixManager(this.config.getString(GobblinClusterConfigurationKeys.HELIX_CLUSTER_NAME_KEY), TestHelper.TEST_HELIX_INSTANCE_NAME,
+            InstanceType.CONTROLLER, zkConnectionString);
 
     this.gobblinYarnAppLauncher = new GobblinYarnAppLauncher(this.config, clusterConf);
   }
 
   @Test
-  public void testCreateHelixCluster() throws Exception {
+  public void testCreateHelixCluster()
+      throws Exception {
     // This is tested here instead of in HelixUtilsTest to avoid setting up yet another testing ZooKeeper server.
-    HelixUtils.createGobblinHelixCluster(
-        this.config.getString(GobblinClusterConfigurationKeys.ZK_CONNECTION_STRING_KEY),
-        this.config.getString(GobblinClusterConfigurationKeys.HELIX_CLUSTER_NAME_KEY));
+    HelixUtils
```

Comment:
formatting seems weird here

Reply:
Yeah, missed that the IDE auto formatted this file. Reverted the autoformat changes.